README.md

```diff
@@ -7,18 +7,62 @@ Related tools, samples, and specs can be found in the [Azure/iot-plugandplay-mod
 
 ## Submit a model
 
+1. Create a GitHub account if you do not have one yet : [https://github.com/join](https://github.com/join).
```

Comment:
nit, rm whitespace before semicolon

Reply:
Updated to use consistent style

[Commit](https://github.com/daisukeiot/iot-plugandplay-models/commit/46d2c845e6a831cd4a039aa2492f3a1e921000a4)